src/plotsbase/atomics.jl

```diff
@@ -100,6 +100,25 @@ end
     )
 end
 
+@default function poly(scene, kw_args)
```

Comment:
What happens to keywords that you don't handle in these `@default` functions?

Reply:
they will get passed through to the backend... in my new branch, i'm warning at a point where i can be sure that the additional kw args are not used!